CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Release date: TBD
  [#512](https://github.com/mattermost/desktop/issues/512)
  - Added the feature to open the application via `mattermost://` link.
  [#616](https://github.com/mattermost/desktop/pull/616)
+ - Change to use Mattermost's own notification sound instead of
+ the built-in tri-tone OSX sound. [#636](https://github.com/mattermost/desktop/issues/636)
```

Comment:
Not super clear here on whether we want issue numbers or pull request numbers here...